apps/calendar/js/app.js

```diff
@@ -304,31 +327,24 @@ module.exports = {
    */
   init: function() {
     debug('Will initialize calendar app...');
-    var self = this;
-    var pending = 2;
-
-    function next() {
-      pending--;
-      if (!pending) {
-        self._init();
-      }
-    }
 
     if (!this.db) {
       this.configure(new Db('b2g-calendar', this), new Router(page));
     }
 
-    // start the workers
-    this.serviceController.start(false);
-
-    navigator.mozL10n.once(next);
     this.db.load(() => {
-      next();
+      this._initControllers();
       // it should only start listening for month change after we have the
       // calendars data, otherwise we might display events from calendars that
       // are not visible. this also makes sure we load the calendars as soon as
       // possible
-      this.store('Calendar').all(() => dayObserver.init());
+      this.store('Calendar').all(() => {
```

Comment:
nothing.. I'll revert to old style.. but during the development I tried a lot of different things here and even had a `window.performance.mark('calendar.all')` here to get a better idea of how long things were taking..

I removed all the `window.performance.mark` calls because I don't know if that would cause any conflicts with datazilla... maybe we should add them back?